ruby/.rubocop.yml

```diff
@@ -16,6 +16,15 @@ Gemspec/RequireMFA:
 Layout/LineLength:
   Max: 200
 
+Metrics/AbcSize:
```

Comment:
Ideally leave these settings in the TODO for us to abstract out later. I probably should have specified more clearly that the purpose of these PR's isn't to fix "everything" just fix "some" things.